cfme/tests/test_rest.py

```diff
@@ -72,22 +72,36 @@ def _finished():
 }
 
 
+COLLECTIONS_IGNORED_57 = {
+    "actions", "alert_definitions", "alerts", "authentications", "configuration_script_payloads",
+    "configuration_script_sources", "load_balancers",
+}
+
+
 @pytest.mark.tier(3)
 @pytest.mark.parametrize(
     "collection_name",
-    ["arbitration_profiles", "arbitration_rules", "arbitration_settings", "automate",
-     "automate_domains", "automation_requests", "availability_zones", "blueprints", "categories",
-     "chargebacks", "cloud_networks", "clusters", "conditions", "container_deployments",
-     "currencies", "data_stores", "events", "features", "flavors", "groups", "hosts", "instances",
-     "measures", "notifications", "orchestration_templates", "pictures", "policies",
-     "policy_actions", "policy_profiles", "providers", "provision_dialogs", "provision_requests",
-     "rates", "reports", "request_tasks", "requests", "resource_pools", "results", "roles",
-     "security_groups", "servers", "service_catalogs", "service_dialogs", "service_orders",
-     "service_requests", "service_templates", "services", "tags", "tasks", "templates", "tenants",
-     "users", "virtual_templates", "vms", "zones"])
+    ["actions", "alert_definitions", "alerts", "arbitration_profiles",
+     "arbitration_rules", "arbitration_settings", "authentications",
+     "automate", "automate_domains", "automation_requests",
+     "availability_zones", "blueprints", "categories", "chargebacks",
+     "cloud_networks", "clusters", "conditions",
+     "configuration_script_payloads", "configuration_script_sources",
+     "container_deployments", "currencies", "data_stores", "events",
+     "features", "flavors", "groups", "hosts", "instances", "load_balancers",
+     "measures", "notifications", "orchestration_templates", "pictures",
+     "policies", "policy_actions", "policy_profiles", "providers",
+     "provision_dialogs", "provision_requests", "rates", "reports",
+     "request_tasks", "requests", "resource_pools", "results", "roles",
+     "security_groups", "servers", "service_catalogs", "service_dialogs",
+     "service_orders", "service_requests", "service_templates", "services",
+     "tags", "tasks", "templates", "tenants", "users", "virtual_templates",
+     "vms", "zones"])
 @pytest.mark.uncollectif(
-    lambda collection_name: (
-        collection_name in COLLECTIONS_IGNORED_56 and current_version() < "5.7"))
+    lambda collection_name:
+        (collection_name in COLLECTIONS_IGNORED_56 and current_version() < "5.7") or
+        (collection_name in COLLECTIONS_IGNORED_57 and current_version() < "5.8")
+)
```

Comment:
The naming here is misleading because `COLLECTIONS_IGNORED_57` for example doesn't really contain collections ignored only in 5.7. If current_version is `5.6` and collection name can be found only in `COLLECTIONS_IGNORED_57`, it will still uncollect.

**REQUIRED:**
They should really be called `COLLECTIONS_ADDED_IN_57` and `COLLECTIONS_ADDED_IN_58` to describe the content correctly. Then it will make sense.